EventFiltering/PWGHF/HFFilterHelpers.h

```diff
@@ -502,6 +543,16 @@ class HfFilterHelper
   float mPtThresholdHighPt3Prongs{8.};                                       // threshold for high pT triggers for 3-prongs
   float mNSigmaTpcKaonFromXicResoToSigmaC{3.};                               // maximum Nsigma TPC for kaons in Xic*->SigmaC-Kaon
   float mNSigmaTofKaonFromXicResoToSigmaC{3.};                               // maximum Nsigma TOF for kaons in Xic*->SigmaC-Kaon
+  o2::framework::LabeledArray<double> mCutsBplus{};                          // selections for B+ candidates (DeltaMass, CPA, DecayLength, ImpactParameterProduct)
+
+  // vertexing parameter
+  bool mPropagateToPCA = true;
+  bool mUseAbsDCA = true;
```

Comment:
`mUseAbsDCA` should be different for charm hadron and beauty hadron (in one case `false`, in the other one `true`). Also maybe there is also no need for data members, I would just put the hard-coded values in the function that you defined with one extra parameter for `useAbsDCA` that can be set `false` for the charm hadron and `true` for the beauty hadron